built front end searchbar component and started search landing page

VictorHLi404 opened the pull request:
will work on the api request later
chekc on figma to see what i think the search page should look like

VictorHLi404 committed:
built searchbar component and began building search landing page

benceruleanlu commented:
Will wait for full feature commit until resolving merge conflicts and building errors.

Left some comments on the Figma.

VictorHLi404 committed:
BUGGED VERSION UPLOADED FOR TEST

reproduce issue by first loading website, search something up, get to search landing page; googles search bar should appear fine, leave back to home page or any related page, search something new again from the home page, get back to search landing page, google search bar is no longer visible but still exists when you look at it with inspect element(?)

VictorHLi404 committed:
Merge branch 'benceruleanlu:main' into main

VictorHLi404 committed:
stable version of search engine

currently linked to my custom version of the search engine, need to transfer versions to the main one + configure the engine to work with articles only

VictorHLi404 commented:
should be stable version, check merge again

VictorHLi404 committed:
patch for broken text updating
